pkg/lib/infra/whatsapp/models.go

```diff
@@ -99,16 +97,15 @@ type UserToken struct {
 type LoginResponseUserExpiresTime time.Time
 
 // Implement Marshaler and Unmarshaler interface
-func (j *LoginResponseUserExpiresTime) UnmarshalJSON(b []byte) error {
-	s := strings.Trim(string(b), "\"")
-	t, err := time.Parse("2006-01-02 15:04:05-07:00", string(s))
+func (j *LoginResponseUserExpiresTime) UnmarshalText(textb []byte) error {
+	t, err := time.Parse("2006-01-02 15:04:05-07:00", string(textb))
 	if err != nil {
 		return err
 	}
 	*j = LoginResponseUserExpiresTime(t)
 	return nil
 }
 
-func (j LoginResponseUserExpiresTime) MarshalJSON() ([]byte, error) {
-	return json.Marshal(time.Time(j))
+func (j LoginResponseUserExpiresTime) MarshalText() ([]byte, error) {
+	return time.Time(j).MarshalText()
```

Comment:
Add a test for this pair of methods. The result of MarshalText should be able to be UnmarshalText. I think this is not now.